### PR TITLE
Trident 18486 - Add parsing patterns

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds a new article:
https://help.sumologic.com/docs/cse/schema/parsing-patterns/

Issue number: [TRIDENT-18486](https://sumologic.atlassian.net/browse/TRIDENT-18486)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
@kimsauce, spellcheck is failing on some text in inline code. Is there a way to have spellcheck ignore all text in inline code? Or to resolve the issue, should I simply add the two items that spellcheck is failing on (`nd` and `wast`) to the `ignore_words_list` in `pr.yml` ?

**Comment by kimsauce** [comment]
> @kimsauce, spellcheck is failing in this PR on text in inline code. Is there a way to have spellcheck ignore all text in inline code? Or to resolve the issue, should I simply add the two items that spellcheck is failing on (`nd` and `wast`) to the `ignore_words_list` in `pr.yml` ?

I'll look into it, but in the meantime can you pls add those two items to the spellcheck ignore list?

File: docs/cse/schema/parsing-patterns.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+---
+id: parsing-patterns
+title: Parsing Patterns
+sidebar_label: Parsing Patterns
```
Comment (kimsauce) [inline docs/cse/schema/parsing-patterns.md:4]:
```suggestion
```

don't need this if the title is the same

**Comment by jpipkin1** [comment]
Adding those words to the `ignore_words_list` in `pr.yml` worked, @kimsauce!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**